dali/operators/reader/nvdecoder/sequencedesc.h

```diff
@@ -29,12 +29,12 @@
 
 namespace dali {
 
-#define SEQUENCEWRAPPER_SUPPORTED_TYPES (float, uint8_t)
+#define SequenceDesc_SUPPORTED_TYPES (float, uint8_t)
 
 // Struct that Loader::ReadOne will read
-struct SequenceWrapper {
+struct SequenceDesc {
```

Comment:
If the data buffer _is_ still here, then I don't think the renaming is warranted.

Reply:
Reverted.